website/src/pages/index.js

```diff
@@ -14,269 +17,144 @@ function getBanner() {
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to={useBaseUrl('guides/getting-started')}>
-                    Learn More
-                  </Link>
-                </div>
-              </div>
-            </div>
-            <div className="col col--4">
-              <div className="card large dark darkest">
-                <div className="card__header">
-                  <h3>dbt Learn</h3>
-                </div>
-                <div className="card__body">
-                  <p>
-                    Learn dbt on your own time with our on demand course or sign up for an upcoming, live public course.
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to="https://learn.getdbt.com">
-                    Learn Now
-                  </Link>
+        <div className="container container--fluid home" style={{ "padding": "0", "background": "#FFF" }}>
+          <Hero heading="Welcome to the dbt Developer Hub" subheading="Your home base for learning dbt, connecting with the community and contributing to the craft of analytics engineering " showGraphic />
+          <section className="resource-section row">
+            <div className="popular-header"><h2>Popular resources</h2></div>
+            <div className="popular-resources">
+              <div className="grid">
+                <div>
+                  <Card
+                    title="What is dbt?"
+                    body="dbt enables data practitioners to adopt software engineering best practices and deploy modular, reliable analytics code."
+                    link="/docs/introduction"
+                    icon="question-mark"
+                  />
+                </div>
+                <div>
+                  <Card
+                    title="Getting started guide"
+                    body="Learn how to set up dbt and build your first models. You will also test and document your project, and schedule a job."
+                    link="/guides/getting-started"
+                    icon="book"
+                  />
+                </div>
+                <div>
+                  <Card
+                    title="Docs"
+                    body="Discover everything dbt has to offer from the basics to advanced concepts."
+                    link="/docs/introduction"
+                    icon="docs"
+                  />
+                </div>
+                <div>
+                  <Card
+                    title="Supported data platforms"
+                    body="dbt connects to most major databases, data warehouses, data lakes, or query engines."
+                    link="/docs/available-adapters"
+                    icon="rocket"
+                  />
```

Comment:
This link 404s when clicking on it. It looks like there's a redirect setup for this which goes to `/docs/supported-data-platforms`. Since it's a SPA, clicking the card doesn't hit the server, causing it to 404. But if you go directly to it, the redirect will fire, redirecting it to `/docs/supported-data-platforms`. Same issue that we ran into in the past for this site.

So updating this link to `/docs/supported-data-platforms` should fix it!

Reply:
Thank you so much!